animation_nodes/nodes/color/c_utils.pyx

```diff
@@ -0,0 +1,27 @@
+# cython: profile=True
+cimport cython
+
+from ... data_structures cimport (
+    ColorList,
+    VirtualColorList,
+    PolygonIndicesList
+)
+
+def loopsColorsList(long Amount, PolygonIndicesList polygons, VirtualColorList colors, 
```

Comment:
No need to pass the amount, you can just get it from the polygons list.

Reply:
Removed the Amount from the inputs.